gateway/src/apicast/policy_chain.lua

```diff
@@ -178,9 +178,13 @@ function _M:add_policy(name, version, ...)
         return self:insert(policy)
 
     elseif err then
+        -- This will only report the last one that failed, but at least users
+        -- can be aware of the issue
+        self.init_failed = true
```

Comment:
This 3 attrs are related, would it make sense to store just one table that contains a `policy_name` and an `error` ?